### PR TITLE
{tools}[system/system] JUBE v2.4.2

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsfc01.int.jusuf.sebastian.cluster - Linux rocky linux 8.4, x86_64, AMD EPYC 7742 64-Core Processor (zen2), 1 x NVIDIA GRID V100-4C, 460.73.01, Python 3.6.8
See https://gist.github.com/e4c184aa6ea5e1e5d5328c89a51f9f19 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/j/JUBE/JUBE-2.4.2.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+sources = [SOURCE_TAR_GZ]
+checksums = ['d1de15e9792802f83521b582d1d144ec81e3d5a28c01dbd945288ea29b946729']
+
+options = {'modulename': 'jube2'}
+
+sanity_check_paths = {
+    'files': ['bin/jube'],
+    'dirs': [],
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/j/JUBE/JUBE-2.4.2.eb:24]: @SebastianAchilles looks like it should be possible to add a simple `sanity_check_command`, `jube --version`?

Comment (bedroge) [reply easybuild/easyconfigs/j/JUBE/JUBE-2.4.2.eb:24]: Simple fix, so I've added this.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/2a870621802a45129cb41757b2745d46 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/0f460c818e62fc5f0ddde52de34129b6 for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**